packages/components/src/dashicon/index.js

```diff
@@ -20,7 +22,17 @@ function Dashicon( { icon, className, ...extraProps } ) {
 		.filter( Boolean )
 		.join( ' ' );
 
-	return <span className={ iconClass } { ...extraProps } />;
+	// For retro-compatibility reasons (for example if people are overriding icon size with CSS), we add inline styles just if the size is different to the default
+	const style =
+		20 !== size
```

Comment:
Should we also consider `"20"` (as string) to be the same as the default?

(No strong opinion on this one, it's just that a lot of people do things like `size="20"` and `size={ 20 }` kind of interchangeably in practice.)

Reply:
I used `!=` to catch both 20 and "20". I added an inline comment to explain that.